### PR TITLE
Add Readme for better understanding of project

### LINKED ISSUE #25: Reformat README
Think would be better if the table includes a column for what the input is &#8212; e.g. `deck` for `print_deck()` &#8212; and include the typical format (and example) of the input as well?

_Originally posted by @maryjess in https://github.com/maryjess/6nimmts/pull/20#discussion_r1744864704_

### PR DESCRIPTION
**Pull request by Shashwat3012**
## Why This Change is Needed:<br>

The enhanced README will help new contributors and users get started more quickly and efficiently by providing them with clear, concise, and well-organized information. It also standardizes the documentation structure, making it easier to maintain in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# 6 Nimmt! Game
```
Comment (maryjess) [inline README.md]: Perhaps can decapitalise the N?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+Welcome to the Python implementation of **6 Nimmt!**, a card game where players try to avoid collecting cards with cattle heads, which represent penalty points. The player with the fewest cattle heads at the end wins!
```
Comment (maryjess) [inline README.md]: Perhaps can add another definition of the game, like `Take 6`?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    - [check_card_in_hand(card, current_deck)](#check_card_in_handcard-current_deck)
+    - [play_game()](#play_game)
+    - [simulate_debug()](#simulate_debug)
+    - [simulate(difficulty)](#simulatedifficulty)
+    - [ask_play_again()](#ask_play_again)
+
+## Game Setup
+
+- **Cards:** 105 cards, each having a unique number and a corresponding number of cattle heads (penalty points).
+  - Cards divisible by 5 have 2 cattle heads (except those divisible by 10, which have 3, and those divisible by 11, which have 5 or 7).
```
Comment (maryjess) [inline README.md]: IMHO, this line could be explained to be more understandable for laymans :)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+## Project Structure
+
+- **Main Python Script:** `main.py`
+  - Implements the core game mechanics.
+  - Contains the setup, distribution, gameplay, and winner determination logic.
+
+## Functions
+
+`print_deck(deck)` :
```
Comment (maryjess) [inline README.md]: Perhaps for the **Functions** section, would it be better to convert to table, containing the function and what they do?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+## How to Play
+
+1. Clone the repository.
+```bash
+git clone https://github.com/maryjess/6nimmts.git
+```
+2. Run the `main.py` file.
+3. Follow the on-screen instructions to play the game.
+
+## License
```
Comment (maryjess) [inline README.md:92]: Would it be possible to display the License in this Repo?

**Review by maryjess — COMMENTED** [review]
Overall, a decent `README` &#8212; good job 👏🏻 

Here are some general/universal reviews so far, for the `README` file:
- Perhaps can change `Added` to `Add` as the title of the PR?
- Perhaps can decapitalise the N? (of *6 Nimmt!* ➡️ *6 nimmt!*)

Will post another review for #16 soon :)

**Comment by Shashwat3012** [comment]
Ok got it! Will submit the new PR by end of the day.

File: README.md (hunk 5 above)
Comment (Shashwat3012) [reply README.md:92]: Hey @maryjess ! Where can I access the license of this repo?

**Comment by Shashwat3012** [comment]
I have made the necessary changes in the latest commit. Please review and let me know if any further change is needed.

File: README.md [hunk 6]
```diff
@@ -1,6 +1,8 @@
-# 6 Nimmt! Game
+# 6 nimmt! Game
```
Comment (maryjess) [inline README.md:1]: Nice!

File: README.md [hunk 7]
```diff
@@ -54,59 +57,28 @@ Welcome to the Python implementation of **6 Nimmt!**, a card game where players
...
-Verifies that a card is in the player's hand.
-
-`play_game()` : 
-Main function to start and manage the game.
-
-`simulate_debug()` : 
-Runs a debug simulation of the game.
+| Function                            | Description                                                                                  |
+|-------------------------------------|----------------------------------------------------------------------------------------------|
+| `print_deck(deck)`                  | Prints the current deck situation.                                                           |
```
Comment (maryjess) [inline README.md:62]: Think would be better if the table includes a column for what the input is &#8212; e.g. `deck` for `print_deck()` &#8212; and include the typical format (and example) of the input as well?

**Review by maryjess — CHANGES_REQUESTED** [review]
Overall, nice changes &#8212; however, imho an update in the table would be better. Thanks a lot!

**Comment by maryjess** [comment]
> Please review and let me know if any further change is needed.

Hi @Shashwat3012 , may I know how's the progress going? :)
Thanks in advance!

File: README.md (hunk 5 above)
Comment (maryjess) [reply README.md:92]: Hi! Sorry I missed this — after some considerations, I think it's not necessary:)

**Comment by maryjess** [comment]
Resolves #25

**Pull request merged**